lib/board.rb

```diff
@@ -0,0 +1,50 @@
+class Board
+	attr_accessor :cells
+	
+	def initialize
+		@cells = Array.new(9," ")
+	end
+	
+	def reset!
+		@cells = @cells.clear

```

Comment:
You don't need to reassign the array after [`#clear`](http://ruby-doc.org/core-2.2.0/Array.html#method-i-clear); `@cells.clear` would have the same effect as `@cells = @cells.clear`.